worlds/metroidprime/Config.py

```diff
@@ -116,7 +117,9 @@ def get_tweaks(world: "MetroidPrimeWorld") -> Dict[str, List[float]]:
 def get_strg(world: "MetroidPrimeWorld") -> Dict[str, List[str]]:
     strg = {**OBJECTIVE_STRG}
     # Set objective text in temple security station
-    objective_text = f"Current Mission: Retrieve {world.options.required_artifacts} Chozo Artifact{'s' if world.options.required_artifacts != 1 else ''}"
+    objective_text = f"Current Mission: Retrieve {
+        world.options.required_artifacts}  Chozo Artifact{
+        's' if world.options.required_artifacts != 1 else ''} "

```

Comment:
```suggestion
        's' if world.options.required_artifacts != 1 else ''}"
```
Not sure whether that space was intentional

Reply:
Definitely was not, thanks for catching 🔥